lib/keycloak/volumes/mgmt-groups.properties

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
these are runtime configs not necessarily volumes. please see https://docs.docker.com/engine/reference/commandline/config/